src/lb/lattices/Lattice.h

```diff
@@ -1148,8 +1149,8 @@ inline static double hsum_double_avx512(__m512d v) {
 									shear_rate += pi[row][column] * pi[row][column];
 								}
 							}
-							shear_rate = sqrt(shear_rate) / (-2.0 * iTau * Cs2 * iDensity);
-							shear_rate /= (1.0 - 1.0 / (2.0 * iTau));
+							shear_rate = sqrt(shear_rate) / (-2.0 * Cs2 * iDensity);
+							shear_rate /= iTau - 0.5;
```

Comment:
I would make this: `shear_rate /= (iTau - 0.5);` to be explicit on its meaning.

Reply:
This division is complementary to the multiplication of (tau - 0.5) in the calculation of the pi tensor.